internal/cli/start.go

```diff
@@ -38,29 +34,30 @@ var networkAPI = []cli.Flag{
 const recoveryAddr = "127.0.0.1:2222"
 
 func Start() error {
+	toolName := "kairos"
```

Comment:
I thought is was better to rename to `kairos` since the actual binary that the users have in the image is also called `kairos` and not `kairos-agent-provider` WDYT?